readme.md

```diff
@@ -49,23 +49,25 @@ Select 1:
 [] Selects all li's which are directly inside a ul of class dropdown (children)
 [] Selects all li's which are anywhere inside a ul of class dropdown (any descendant)
 [] Selects all ul's of class dropdown, as well as the children elements that are li's
-[] Selects all ul's of class dropdown, only if their children are exclusively li's
+[x] Selects all ul's of class dropdown, only if their children are exclusively li's
 ```
 
 ## Question 6
 
 Create a variable and store the string "pizza" in it
 
 ```js
-answer goes here
+var pizza = ["pizza"];
 ```
 
```

Comment:
this is correct technically, but why did you choose to make an array with one item?